<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="body_layout">
     <item>
      <widget class="QPushButton" name="autotrade_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>시작</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="transaction_table">
       <property name="styleSheet">
        <string notr="true">QHeaderView::section { background-color: rgba(189,189,189,100) }</string>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>날짜</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color alpha="200">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>유형</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color alpha="200">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>심볼</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color alpha="200">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>가격</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color alpha="200">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color alpha="200">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autotrade_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>push_trade_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>-33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>push_trade_button()</slot>
 </slots>
</ui>
